### ISSUE TITLE
Explaining loss/accuracy curve behavior when resuming training...

### ISSUE THREAD

**Issue by Mahi-Mai** [keras-team/keras#13769]
**System information**  
- Have I written custom code (as opposed to using example directory): 
yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): 
 Linux
- TensorFlow backend (yes / no):  
yes
- TensorFlow version:  
tensorboard==1.14.0
tensorflow-estimator==1.14.0
tensorflow-gpu==1.14.0
- Keras version:  
Keras==2.2.4
Keras-Applications==1.0.7
Keras-Preprocessing==1.0.9
- Python version:  3.6
- CUDA/cuDNN version:  
- GPU model and memory:  Tesla K80 x 8

**Describe the current behavior**
I'm training a model in iterations.  So, say I'll halt training at 15 epochs, save the model weights, and examine the model's behavior.  If the model "passes" I start another training job, say for 20 epochs.  I resume where I left off by reinstantiating the model and loading the saved weights.

However, when I do this I see a pair of spikes in my loss/accuracy curves before they settle back into their expected behavior.  In the example below, the red line is where training was halted and resumed.

<img width="703" alt="Screen Shot 2020-02-13 at 10 31 21 AM" src="https://user-images.githubusercontent.com/17129394/74459078-50502000-4e50-11ea-894c-02424f26d512.png">

**Describe the expected behavior**
When reinstantiating the model with the old weights, I would expect the curve to be continuous.  Is it because I'm loading weights into a "fresh" model, so the model "explores" a bit before settling back into its groove?

I'm just trying to understand it better.

**Comment by Saduf2019** [comment]
@Mahi-Mai 

We see that you are using old version of tensorflow which is not actively supported now, please verify on 2.x and open a new issue in case you face any errors, hence moving this to closed status.
